### DISCUSSION TITLE
Is there a way in IdeaVim to record a macro that edits multiple files

### DISCUSSION THREAD

**Discussion by VasoTodorovic** [JetBrains/ideavim#809] [Q&A] [answered]
I want to record macro that:
viwy   :e file2 p ctr-o w
I am new to IdeaVim and even vim in general. But does IdeaVim record macro just in one file?
Is there option to record such macro that edit multiple files in IdeaVim?

**Comment by lippfi** [comment, marked as answer]
It's not currently possible. Macro execution is bound to a specific editor.

However, there is a refactoring in progress that aims to resolve some issues with the editor binding. I cannot provide an estimate for when the issue will be adressed, but work is in progress.
